jupyter_server/auth/identity.py

```diff
@@ -81,6 +81,39 @@ def fill_defaults(self):
         if not self.display_name:
             self.display_name = self.name
 
+    def to_string(self) -> str:
+        """Serialize a user to a string for storage in a cookie
+        If overriding in a subclass, make sure to define from_string as well.
+        Default is just the user's username.
+        """
+        # default: username is enough
```

Comment:
I don't understand how this comment comes into play here.  Is the help string and comment valid?

Reply:
Comment and docstring are out of date, behavior changed in [#863](https://github.com/jupyter-server/jupyter_server/pull/863/files#r933202654).